libs/dm.json

```diff
@@ -1,5 +1,5 @@
 [
+  {"name":"Byond","id":"dm.byond","tech":["app-arch"],"tags":["oop"],"repo":"","imports":[""],"examples":"https://github.com/tgstation/tgstation","status":"awaiting-model"},
```

Comment:
No repo for this? Is there a way to get source code for byond?

Reply:
That is not mine, verification.js did it. Probably in some previous MR someone did not run it changes have been accepted.

Reply:
@asurkov could you consider adding a repo check to your verification script? No repo means we can't train a classifier.

Reply:
> @asurkov could you consider adding a repo check to your verification script? No repo means we can't train a classifier.

it has examples, so we should be able to train classifier, right? currently verification tool fails when neither repo nor examples is provided.